src/components/relationships-panel/package.json

```diff
@@ -0,0 +1,11 @@
+{
+  "name": "@bcrs-shared-components/relationships-panel",
+  "version": "0.0.0",
+  "publishConfig": {
+    "access": "public"
+  },
+  "dependencies": {
+    "@bcrs-shared-components/enums": "^1.0.26",
```

Comment:
You need to rebase. This is up to v1.0.27 now.

Reply:
Yes, I'm already on it!

Reply:
FYI, when I run the Lerna commands for you, it will auto-update this to 28 since Lerna will also be concurrently publishing the enums package.